app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -81,6 +82,13 @@ def handle_valid_otp
     reset_otp_session_data
   end
 
+  def handle_valid_piv_cac
+    handle_valid_otp_for_authentication_context
+
+    redirect_to after_otp_verification_confirmation_url
+    reset_otp_session_data
+  end
```

Comment:
What do you think about moving this method into `piv_cac_verification_controller.rb` since it only applies to that controller?

Reply:
That makes sense. The other option is to add a parameter to the more generic version to manage remember device since the only reason for making this on its own was to disallow remember device for piv/cac. I only put it here because the other, similar method was here.